### ISSUE TITLE
Preview menu: arrows navigation doesn't work

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#24453]
**Describe the bug**
The WordPress 5.5 block editor will ship with a new "Preview" menu with options for desktop, tablet, mobile, and traditional preview in a new browser tab. Since it's an ARIA menu and looks like other menus where arrows navigation works, I'd expect  arrows navigation to work on this menu as well. It doesn't.

**To reproduce**
- edit a post
- use the keyboard
- tab to the Preview menu button
- press Enter to open the menu
- the menu opens and focus is moved to the first menu item: so far so good
- use the Down and Up arrow keys to try to navigate through the menu items
- nothing happens

Additionally:
- open your browser's dev tools or shrink the browser's window height so that the page has some scrolling
- observe that while using the arrow keys on the menu items, the page (see the admin menu) scrolls, which means the default action is not prevented

Animated GIF:

![preview menu arrowing](https://user-images.githubusercontent.com/1682452/89730589-3d558b80-da40-11ea-800c-6c51d130e154.gif)

**Expected behavior**
Arrows navigation on the Preview menu items to work and the default action to be prevented.

**Editor version (please complete the following information):**
- WordPress version: 5.5 Beta 2
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Tested both
- If the Gutenberg plugin is installed, which version is it? 8.7.0 latest master

Noting that, technically, this is not a regression but it's a new feature introduced in WordPress that doesn't meet the expected accessibility best practices.

**Comment by ntsekouras** [comment]
Thanks for reporting this @afercia!
